lib/ex_ex/enum.ex

```diff
@@ -16,4 +16,7 @@ defmodule ExEx.Enum do
     Enum.map(enumerable, &Task.async(fn -> fun.(&1) end))
     |> Enum.map(&Task.await/1)
   end
+
+  def has_key?(map, key) when is_map(map), do: Map.has_key?(map, key)
+  def has_key?(keyword, key) when is_list(keyword), do: Keyword.has_key?(keyword, key)
```

Comment:
It feels like the `keyword` param should be plural here since it's a list. I'd be okay with using `enum` to represent the enumerable in both of them, fwiw.

Reply:
`enum` works for me